### ISSUE TITLE
error after running docker-compose command

### ISSUE THREAD

**Issue by pallasite99** [pezzolabs/pezzo#12]
```console
C:\Users\Salil_Apte\Documents\Development\pezzo>docker-compose up
[+] Running 0/2
 - pezzo-server Error                                                                                              3.3s
 - pezzo-console Error                                                                                             3.3s
Error response from daemon: pull access denied for pezzolabs/pezzo-console, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

Not being able to proceed further. Guessing the console repo is private 👶

**Comment by arielweinberger** [comment]
Thanks for reporting! The fix should be straightforward and I'll push it as soon as I get home.

In the meantime, you can adjust the images in the Docker compose file to:

ghcr.io/pezzolabs/pezzo/server

And

ghcr.io/pezzolabs/pezzo/console

If that worked, feel free to submit a PR :)

**Comment by pallasite99** [comment]
Cool, I'll see if it works locally first

**Comment by pallasite99** [comment]
```console
C:\Users\Salil_Apte\Documents\Development\pezzo>docker login
Login with your Docker ID to push and pull images from Docker Hub. If you don't have a Docker ID, head over to https://hub.docker.com to create one.
Username: xxx
Password:
Login Succeeded

Logging in with your password grants your terminal complete access to your account.
For better security, log in with a limited-privilege personal access token. Learn more at https://docs.docker.com/go/access-tokens/

C:\Users\Salil_Apte\Documents\Development\pezzo>docker-compose up
[+] Running 0/2
 - pezzo-console Error                                                                                             5.6s
 - pezzo-server Error                                                                                              5.6s
Error response from daemon: pull access denied for pezzolabs/pezzo-server, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

seems something else is missing. Maybe forcing docker to pull image from ghcr?

**Comment by pallasite99** [comment]
Think its `pezzolabs/pezzo/console`

**Comment by pallasite99** [comment]
Facing the same error as yesterday. It seems I've not installed something but don't know what is missing. Had taken latest pull:

![image](https://github.com/pezzolabs/pezzo/assets/26508636/f374718e-9706-479f-b9c4-f692633b528e)

**Comment by pallasite99** [comment]
Yup I had followed the same :)

But its throwing this issue since yesterday

**Comment by arielweinberger** [comment]
Could you try running the following and then trying again:

npx nx graphql-codegen graphql

And then trying to run the console?

**Comment by arielweinberger** [comment]
@pallasite99 I have tried going through a fresh installation and it all worked just fine. I found a few quirks in the docs and fixed it on PR #13, but nothing crucial that should prevent you from running the app.

Do you mind trying again? Please keep me posted.

**Comment by pallasite99** [comment]
Working fine now. The docker issue being a minor one, I missed the steps in the doc and thought that we needed to perform options 1 and 2 together. Was able to view the console UI, hence closing this issue, thanks!
